src/Components/Home/FeaturesAndBlogs.tsx

```diff
@@ -57,9 +34,15 @@ export default function HomeSection4() {
                 </div>
             </div>
             <div className="blogs">
-                {blogs.map((blog: any, index: number) => (
-                    <BlogCard obj={blog} key={index} />
-                ))}
+                {blogs.map((blog: bloginfo, index: number) => {
+                    if (index < 2) {
+                        return (
+                            <BlogCard obj={blog} key={index} />
+                        )
+                    }
+                    return null;
```

Comment:
bad implementation of retrieving the first two array elements

Reply:
this has been fixed